InferenceSystem/README.md

```diff
@@ -69,6 +70,29 @@ setx AZURE_COSMOSDB_PRIMARY_KEY "<yourprimarykey>"
 export AZURE_COSMOSDB_PRIMARY_KEY="<yourprimarykey>"
 ```
 
+## Get connection string for interface with App Insights
+
+Go to the [Azure portal](https://portal.azure.com/)
+
+Go to the `"LiveSRKWNotificationSystem"` resource group and within that go to the `"orcanotification"` App Insights service
```

Comment:
Please create a new app insights. The old one is already collecting info for azure functions.

Note that I found out because the dashboard wasn't showing 0 as expected. You'll have to update the query for the dashboard as well.